demo/src/main/java/com/google/android/fhir/demo/FhirApplication.kt

```diff
@@ -43,7 +43,7 @@ class FhirApplication : Application() {
         ServerConfiguration("https://hapi.fhir.org/baseR4/")
       )
     )
-    Sync.oneTimeSync<FhirPeriodicSyncWorker>(this)
+    Sync.oneTimeSync(this, FhirPeriodicSyncWorker::class.java)
```

Comment:
```suggestion
    Sync.oneTimeSync<FhirPeriodicSyncWorker>(this)
```
Revert this change as the  Sync.oneTimeSync signature shouldn't change.